src/az390.java

```diff
@@ -2815,8 +2823,12 @@ private void process_bal_op(){
     	loc_len = 6;
     	get_hex_op(1,2);
     	get_hex_bdddhh2();
-    	skip_comma();
-    	get_hex_byte();
+		if (!bal_abort && exp_next_char(',')){ // DSH #414 z16 add support for EB71 LPSWEY D1(B1) witn no second opr
+      	   skip_comma();
+    	   get_hex_byte();
+		} else {
+		   get_hex_zero(2);
+		}
```

Comment:
First off, I make no claim as to understanding this code. 

Old code did skip_comma(), get_hex_byte(), then replaced obj_code with 3 substrings.

New code added the "if else" before the replacment of obj_code. The "else" is taken when bal_abort is true or the next claracter in the expression is not a comma, concatenating 2 bytes of "0" to obj_code. Now the replacement of obj_code is done.

Just being sure this is what you intended.

Same comment applies below for your new lines 3042-3048.